ddtrace/internal/telemetry/data/application.py

```diff
@@ -0,0 +1,45 @@
+import platform
+from typing import Optional
+from typing import TypedDict
+
+from ....settings import _config as config  # noqa: E402
+from ....version import get_version
+
+
+Application = TypedDict(
+    "Application",
+    {
+        "service_name": str,
+        "service_version": str,
+        "env": str,
+        "language_name": str,
+        "language_version": str,
+        "tracer_version": str,
+        "patches": str,
+        "runtime_name": str,
+        "runtime_version": str,
+    },
+)
+
+_application = None  # type: Optional[Application]
+
+
+def get_application():
+    # type: () -> Application
+    global _application
+
+    if _application:
+        return _application
+
+    _application = {
+        "service_name": config.service or "unnamed_python_service",
+        "service_version": config.version or "DD_SERVICE_VERSION NOT SET",
+        "env": config.env or "DD_ENV NOT SET",
```

Comment:
we should default these to just empty strings to indicate they were not set

Reply:
The service_name, service_version, and env field must be set by the client for the intake service to accept the payload. We can use empty strings here. We'll need to set some default value